vault/resource_gcp_auth_backend_test.go

```diff
@@ -32,7 +32,7 @@ const gcpJSONCredentials string = `
 `
 
 func TestGCPAuthBackend_basic(t *testing.T) {
-	testutil.SkipTestEnvSet(t, testutil.EnvVarSkipVaultNext)
```

Comment:
This test no longer needs this skip step since all features being tested have been released. Since I was in the code I went ahead and removed it